src/sentry/digests/backends/redis.py

```diff
@@ -130,11 +130,11 @@ def add(
                     "ADD",
                     self.namespace,
                     self.ttl,
-                    timestamp,
+                    int(timestamp),
```

Comment:
Why do we need to cast this float to int?

Reply:
https://www.lua.org/manual/5.4/manual.html

> math.randomseed ([x [, y]])
> When called with at least one argument, the integer parameters x and y are joined into a 128-bit seed that is used to reinitialize the pseudo-random generator; equal seeds produce equal sequences of numbers. The default for y is zero. 

DragonflyDB use lua 5.4. DragonflyDB it's the only way I found to have a high available cluster compatible with Sentry. There is multiple issue open about redis cluster in Sentry because of both library RedisCluster and Redis Blaster. 

But if there is an other way I will be interested

Reply:
I'm not sure if this is a breaking change or not. I recommend creating a test to ensure float to int precision loss is handled properly.

Reply:
there is some test that failed because of precision loss. So I'm not sure about the impact for existing data. 
![image](https://github.com/getsentry/sentry/assets/4802105/2e453678-04b6-4b96-af9e-d01a9517ec7e)

Reply:
I simplify the the change, so test should be good